src/operations/invoke_host_function.js

```diff
@@ -27,29 +34,18 @@ export function invokeHostFunction(opts) {
  *
  * @param {object} opts - options for the operation
  * @param {xdr.HostFunction[]} opts.functions - a list of contract functions to
- *    invoke in this operation. each item should contain an object with a set of
- *    args and an optional list of auths (like those required by
- *    {@link Operation.invokeHostFunction}).
+ *    invoke in this operation. 
  * @param {string} [opts.source] - an optional source account
  *
  * @returns {xdr.Operation} an Invoke Host Function operation
  *    (xdr.InvokeHostFunctionOp) with xdr.HostFunction instances corresponding
  *    to each invocation
  *
- * @warning This function does not support setting a different source acco
+ * @warning This function does not support setting a different source account.
  */
 export function invokeHostFunctions(opts) {
-  const functions = opts.functions.map((hostFn) => {
```

Comment:
the `functions` input parameter here is already typed to `xdr.HostFunction` so the `args` are enforced through the class definition. This manual 'args' validation is retained on the the singular invoke contract function we've kept in place `invokeHostFunction` since that only takes on object as input. let me know if this does not line up?

Reply:
Ahh okay, I see it now. We def need some higher-level abstractions around these, but that's for another time.